index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:title" content="판다마켓">
+    <meta property="og:description" content="일상의 모든 물건을 거래해보세요">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://capable-donut-a02c71.netlify.app/">
+    <meta property="og:image" content="image/preview_img">
```

Comment:
카카오톡 기준으로 og:image가 나오지 않네요. 이미지 참조를 문제같으니 경로를 수정하시면 될거같아요

```suggestion
    <meta property="og:url" content="https://capable-donut-a02c71.netlify.app/">
    <meta property="og:image" content="./image/preview_img">
```

<img width="338" alt="스크린샷 2024-04-02 오후 6 42 58" src="https://github.com/codeit-bootcamp-frontend/6-Sprint-Mission/assets/42806386/dc4fcf93-3e2e-4b27-bd77-a221e3a089d3">